test/e2e/lib/components/secure-payment-component.js

```diff
@@ -309,7 +309,7 @@ export default class SecurePaymentComponent extends AsyncBaseContainer {
 
 		const cartElement = await this.driver.findElement( this.getCartTotalSelector() );
 
-		const cartText = await cartElement.getText();
+		const cartText = await cartElement.getAttribute( 'innerText' );
```

Comment:
I ran the test in headful mode a couple of times and verified that this attribute was visible. I'm not sure why `getText` was failing, but it might be related to https://stackoverflow.com/questions/20888592/gettext-method-of-selenium-chrome-driver-sometimes-returns-an-empty-string (See: p1605828913144100-slack-C1A1EKDGQ?thread_ts=1605826407.143600&cid=C1A1EKDGQ).